docs/tasks/manage-daemon/rollback-daemon-set.md

```diff
@@ -0,0 +1,138 @@
+---
+assignees:
+- janetkuo
+title: Performing a Rollback on a DaemonSet
+---
+
+{% capture overview %}
+
+This page shows how to perform a rollback on a DaemonSet. 
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+* The DaemonSet rollout history and DaemonSet rollback feature are only
+  supported in `kubectl` in Kubernetes version 1.7 or later.
+* Make sure you know how to [perform a rolling update on a
+  DaemonSet](/docs/tasks/manage-daemon/update-daemon-set/)
+
+{% endcapture %}
+
+
+{% capture steps %}
+
+## Performing a Rollback on a DaemonSet
+
+1. Find the DaemonSet revision you want to roll back to:
```

Comment:
Should I use 

1. 
1. 
1. 

or

### Step 1:
### Step 2:
### Step 3:
?